llarp/router/router.cpp

```diff
@@ -874,6 +874,60 @@ namespace llarp
     m_LastStatsReport = now;
   }
 
+  std::string
+  Router::status_line()
+  {
+    std::string status;
+#if defined(WITH_SYSTEMD)
```

Comment:
Should nuke this ifdef guard; this status line doesn't depend on systemd anymore.

Reply:
done